Sources/BuildTimeEval/configure.swift

```diff
@@ -0,0 +1,35 @@
+//===----------------------------------------------------------------------===//
+// Copyright © 2025 Apple Inc. and the Pkl project authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//===----------------------------------------------------------------------===//
+
+import Gen
+import PklSwift
+import Vapor
+
+// configures your application
+public func configure(_ app: Application) async throws {
+    let conf = try! PklDecoder.decode(
+        Config.Module.self, from: [UInt8](Data(contentsOf: URL(filePath: "pkl/dev/config.msgpack")))
+    )
```

Comment:
Per my above suggestion of bundling the pkl-binary, this should change to:

```suggestion
    guard let fileURL = Bundle.module.url(forResource: "config", withExtension: "msgpack") else {
        fatalError("Expected `config.msgpack` to be bundled, but it was not")
    }
    let conf = try! PklDecoder.decode(
        Config.Module.self, from: [UInt8](Data(contentsOf: fileURL))
    )
```